### PR TITLE
Proposing an elitist CMA for inoculation

### PR DESCRIPTION
**Pull request by teytaud**
Part of https://github.com/facebookresearch/nevergrad/issues/847

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

For hard problems with excellent inoculation this should be useful.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/optimization/optimizerlib.py [hunk 1]
```diff
@@ -396,6 +402,7 @@ def __init__(
 
 
 CMA = ParametrizedCMA().set_name("CMA", register=True)
+ECMA = ParametrizedCMA(elitist=True).set_name("ECMA", register=True)
```
Comment (jrapin) [inline nevergrad/optimization/optimizerlib.py:405]: is it needed here? I would tend to move ECMA (and FCMA by the way) to experimentalvariants

File: nevergrad/optimization/optimizerlib.py [hunk 2]
```diff
@@ -367,6 +370,8 @@ class ParametrizedCMA(base.ConfiguredOptimizer):
     ----------
     scale: float
         scale of the search
+    elitist: bool
+        whether we switch to elitist mode
```
Comment (jrapin) [inline nevergrad/optimization/optimizerlib.py:374]: can you explain in the docstring what is elitist mode?

Comment (teytaud) [reply nevergrad/optimization/optimizerlib.py]: Done.

File: nevergrad/optimization/optimizerlib.py (hunk 1 above)
Comment (teytaud) [reply nevergrad/optimization/optimizerlib.py]: As you prefer; done.

Comment (teytaud) [reply nevergrad/optimization/optimizerlib.py]: I keep FCMA actually due to a weird dependency in mix-simulator.

**Review by jrapin — APPROVED** [review]

**Pull request merged**